### ISSUE TITLE
TS1015: Parameter cannot have question mark and initializer

### ISSUE THREAD

**Issue by joebowbeer** [Khan/flow-to-ts#134]
The question mark should be dropped in conversion:

```
// @flow
function foo(flag?: boolean = true) {};
```

Bad:

```
function foo(flag?: boolean = true) {}
```

The optional flag? type is boolean | undefined but the initializer removes the undefined, leaving:

```
function foo(flag: boolean = true) {}
```

https://flow-to-ts.netlify.com/#code=Ly8gQGZsb3cKZnVuY3Rpb24gZm9vKGZsYWc/OiBib29sZWFuID0gdHJ1ZSkge307&prettier=1&semi=1&singleQuote=0&tabWidth=4&trailingComma=all&bracketSpacing=0&arrowParens=avoid&printWidth=80&inlineUtilityTypes=0

**Comment by k4b7** [comment]
TIL.  Thanks for the bug report.

**Comment by gomesalexandre** [comment]
I'm still experiencing this one on latest version of flow-to-ts 😢

**Comment by SethDavenport** [comment]
As am I. Looks like no release has been published to npm since the fix was merged?

**Comment by gomesalexandre** [comment]
> As am I. Looks like no release has been published to npm since the fix was merged?

Seems like it. 
It's fairly trivial to fix, during our conversion we had to run multiple homemade codemods anyway, after the `flow-to-ts` run, as `flow-to-ts` still left us with a lot of errors (flow vs. TS react primitives, flow class properties etc) 

Something like [this](https://gist.github.com/gomesalexandre/d52c1ae13db4622397f4f91c75c12f93) should do the trick.

**Comment by SethDavenport** [comment]
Thanks - I ended up just installing from github directly instead of using an npm release: `yarn global add git+ssh://git@github.com:Khan/flow-to-ts.git#8fd8a511d4d2a7366a604b0381c1cf6950d3006b`

**Comment by k4b7** [comment]
I've published 0.1.7 and it includes the fix for this issue.
